DDDEastAnglia/DataAccess/DefaultSponsorSorter.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DDDEastAnglia.DataAccess.SimpleData.Models;
+
+namespace DDDEastAnglia.DataAccess
+{
+    public static class SponsorExtensions
+    {
+        public static IEnumerable<Sponsor> OrderBySponsorSorter(this IEnumerable<Sponsor> sponsors)
```

Comment:
The method name implies that it could take different `SponsorSorter`s, but it does not; as a result, it's not very Open-Closed. I think I'd also prefer the `SponsorSorter` be given to the `SponsorService` (or whatever) so that it is able to provide a sorted list of sponsors, on a flexible basis, but that's possibly a stylistic thing.